weather-app/app/components/Sidebar.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import styles from "./Sidebar.module.scss";
+
+const Sidebar = () => {
+  const [isWeatherActive, setIsWeatherActive] = useState<boolean>(true);
+  const [isFavoriteListActive, setIsFavoriteListActive] =
+    useState<boolean>(false);
+
+  const weatherIconHandler = () => {
+    if (!isWeatherActive) {
+      setIsWeatherActive(true);
+      setIsFavoriteListActive(false);
+    }
+  };
+
+  const favoriteListIconHandler = () => {
+    if (!isFavoriteListActive) {
+      setIsWeatherActive(false);
+      setIsFavoriteListActive(true);
+    }
+  };
+
+  return (
+    <div className={styles.sidebar}>
+      <Link className={styles.link} href={"/"} onClick={weatherIconHandler}>
```

Comment:
You can remove `onClick={weatherIconHandler}` here and write like below.
`<Link className={styles.link} href={"/"}>`